From 7bd951d4d234ec1e408496196df02193d1e236ab Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 12 Jul 2021 15:15:44 +0100
Subject: [PATCH 0829/2201] ARM: dts: bcm2711: fold in the correct interrupt

The new vec node in bcm2711.dtsi should have the correct interrupt
number to start with, rather than include the bcm283x version and
patch it later.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711.dtsi | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711.dtsi b/arch/arm/boot/dts/bcm2711.dtsi
index ef78c6e5f437..b9edf0dee35c 100644
--- a/arch/arm/boot/dts/bcm2711.dtsi
+++ b/arch/arm/boot/dts/bcm2711.dtsi
@@ -304,7 +304,7 @@
 			compatible = "brcm,bcm2711-vec";
 			reg = <0x7ec13000 0x1000>;
 			clocks = <&clocks BCM2835_CLOCK_VEC>;
-			interrupts = <2 27>;
+			interrupts = <GIC_SPI 123 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
 		};
 
@@ -1106,7 +1106,3 @@
 &usb {
 	interrupts = <GIC_SPI 73 IRQ_TYPE_LEVEL_HIGH>;
 };
-
-&vec {
-	interrupts = <GIC_SPI 123 IRQ_TYPE_LEVEL_HIGH>;
-};
-- 
2.17.1

